text.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
```

Comment:
If you don't need this file you can remove it to. I checked it and it looks similar to `index.html`

Reply:
OKay, I removed it